chapters/hpmor-chapter-097.tex

```diff
@@ -295,7 +295,7 @@
 
 Lucius Malfoy turned his gaze to Harry Potter then, and his eyes simmered in pure grey fury.
 
-“If you have deceived me,” Lucius Malfoy said in tones of low anger, “if all this is a lie, then I will not forgive. But if this is not deception… Bring me the proof to convict Dumbledore of this murder before the Wizengamot, or evidence enough to have him cast down, and there is nothing that House Malfoy will not do for you, Harry Potter. Nothing.”
+“If you have deceived me,” Lucius Malfoy said in tones of low anger, “if all this is a lie, then I will not forgive. But if this is not deception…Bring me the proof to convict Dumbledore of this murder before the Wizengamot, or evidence enough to have him cast down, and there is nothing that House Malfoy will not do for you, Harry Potter. Nothing.”
```

Comment:
I think this is the end of a sentence, so add a full stop after the ellipsis rather than removing a space.